EmojicodeCompiler/Lexer.cpp

```diff
@@ -133,7 +134,7 @@ TokenStream lex(const char *path) {
                     }
                     continue;
                 case TokenType::Variable:
-                    if (detectWhitespace(c, &sourcePosition.character, &sourcePosition.line)) {
+                    if ((c != 0xFE0F) && detectWhitespace(c, &sourcePosition.character, &sourcePosition.line)) {
```

Comment:
If I do that, it introduces a bunch of bugs, going as far as to break the ’s’ package. The check will have to remain here.